composer.json

```diff
@@ -31,7 +31,8 @@
         {"name": "Jonathan Wage", "email": "jonwage@gmail.com"}
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^7.3 || ^8",
+        "ext-pdo": "*",
```

Comment:
This should be reverted. The change in the first line is non-essential, the addition of the second line is wrong.

Reply:
Wrong for 3.0 or does this also affect 2.12?

Reply:
Only wrong for `3.0.x`.